kedro-datasets/tests/pandas/test_deltatable_dataset.py

```diff
@@ -66,7 +66,7 @@ def test_append(self, deltatable_dataset_from_path, dummy_df):
         """Test saving by appending new data."""
         deltatable_dataset_from_path.save(dummy_df)
         new_df = pd.DataFrame({"col1": [0, 0], "col2": [1, 1], "col3": [2, 2]})
-        appended = pd.concat([dummy_df, new_df], ignore_index=True)
+        appended = pd.concat([new_df, dummy_df], ignore_index=True)
```

Comment:
Is this the correct behavior? I recalled there is a version change and the order get flipped.

Reply:
There's a change between `deltalake` 0.15.1 and 0.15.2 which is causing the order of the saved dataframe to be flipped, I needed to unpin `deltalake` to unpin `pyarrow` 😓

Reply:
The next datasets release is a breaking one anyway (`3.0.0`) so this change is fine IMO.